code/modules/clothing/clothing.dm

```diff
@@ -175,6 +175,8 @@ BLIND     // can't see anything
 	slowdown = SHOES_SLOWDOWN
 	var/blood_state = BLOOD_STATE_NOT_BLOODY
 	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0,BLOOD_STATE_XENO = 0, BLOOD_STATE_OIL = 0, BLOOD_STATE_NOT_BLOODY = 0)
+	var/can_hold_knives = 0//if set to 1, the shoe can hold knives and edaggers

```

Comment:
not really, it's just an object var on a parent type object. same type of stuff is all over the place, like on energy guns self-charge var